### DISCUSSION TITLE
package versions could not be found in R

### DISCUSSION THREAD

**Discussion by manuelatito** [teamalgoritma/community#873] [Q&A] [answered]
Hi.. saya ingin bertanya, ketika saya membuka Rproj dari kelas muncul notif seperti ini, apa yg harus saya lakukan? apabila ada package yg harus diupdate bgm caranya? atau notif ini bisa diabaikan saja? Terima kasih
![Github R (2)](https://github.com/teamalgoritma/community/assets/164650405/e1a87bf0-69ea-42f6-af20-cd0da981a890)
![Github R (3)](https://github.com/teamalgoritma/community/assets/164650405/ba40c8e1-099f-4e7a-b69e-40d25403cf33)
![Github R](https://github.com/teamalgoritma/community/assets/164650405/965421eb-1f06-4897-8000-e8a442fa87ac)

**Comment by ramdhinafinita** [comment]
Dear @manuelatito 
Berdasarkan error yang dilampirkan, kemungkinan memang tidak terdapat packages yang sesuai dengan R pada Laptop Anda atau memang ada pada laptop anda, namun CRAN sudah melakukan archive pada pacagkaes yang sudah lama tersebut. 
Anda bisa mencoba untuk install/update packages yang dimaksud dengan langkah berikut:
![image](https://github.com/teamalgoritma/community/assets/64061969/875cc834-bd86-4d64-82a9-c114652737f7)

Semoga berhasil

**Reply by manuelatito** [reply, marked as answer]
Dear kak @ramdhinafinita,
Terima kasih atas jawabannya kak, akan saya coba sesuai arahan kk..
